### PR TITLE
Marked OracleConfig as deprecated, removed tests

### PR DESCRIPTION
**Pull request by PaulWay**
This parser is already removed in the master branch, so it is marked as deprecated in the 1.x branch.

### CODE REVIEW

**Comment by bfahr** [comment]
@PaulWay I recommend leaving tests in place until function is removed.

File: insights/parsers/oracfg.py [hunk 1]
```diff
@@ -71,6 +72,9 @@ class OracleConfig(Parser):
             string
 
     """
+    def __init__(cls, *args, **kwargs):
```
Comment (bfahr) [inline insights/parsers/oracfg.py]: I expected to see `self` here instead of `cls`.

Comment (xiangce) [reply insights/parsers/oracfg.py]: Hi @PaulWay - How do you think about the comments that Bob commented above?

**Comment by PaulWay** [comment]
OK, I've fixed those issues - let me know what you think!

**Comment by xiangce** [comment]
@PaulWay - I have to revert this PR. Since I have merged the #819 already, the test breaks.
Please update this parse and re-submit it in a new PR. Sorry for that.

**Comment by PaulWay** [comment]
Hi @xiangce OK, try #823 instead.

**Pull request merged**